modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/view_flat_users.jsp

```diff
@@ -29,6 +29,7 @@ String displayStyle = viewFlatUsersDisplayContext.getDisplayStyle();
 %>
 
 <clay:management-toolbar
+	itemsType='<%= LanguageUtil.get(request, "users") %>'
```

Comment:
Aren't we already translating this [here](https://github.com/liferay-frontend/liferay-portal/pull/3058/files/013e1bf7d59d41c576d37a9a450a3fc374dfdbd8#diff-693ec17285b9e3a9bd29a88403e0b1a280ca0c79d47868883f93ddaa8d19d47eR668)? Do tag users need to translate the text in `itemType`, or the tag takes care?

Also, probably not related to this particular ticket: do we know the places where we need to pass the itemsType across DXP? Are there plans to check the existing usages?

Reply:
Hi @dsanz! we translate the itemType here, the tag is only responsible for translating the words that are not unknowns here `x-of-x-x-selected`.

Regarding passing `itemType` through DXP. `itemType` has been initialized to `Items`, so it won't affect other places where this taglib is used. I mean, by default it will read `x of x items selected`. Therefore there is no plan to verify these usages. Each team can update this if they see it necessary.

Reply:
@dsanz is right, we're double translating here. This attribute probably should follow the same pattern of the rest of the tags an be translated by the tag (titles are being translated by the tag, labels, etc). If there're cases where they don't want it to be localize we could rise an update then (we have a case right now), but it's a weird one that is better to solve by demand.

Also, I'm not sure if we're taking care of the SSR. Are we adding this/those `aria-label` in the java markup?

Reply:
@dsanz @carloslancha I have uploaded a [last commit](https://github.com/liferay-frontend/liferay-portal/pull/3058/commits/70911b0b94bf91121686caed9b36c1df88e85edd) adding/updating aria-labels and labels for the SSR and solving the problem with the double translation. Could check this? Thanks a lot!